repositories.bzl

```diff
@@ -15,6 +15,8 @@
 ################################################################################
 #
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load(":protobuf.bzl", "protobuf_repositories")
```

Comment:
Can you get rid of local protobuf copy as well? It's getting out of sync and will lead to issues sooner or later.

Reply:
done.